src/components/shared/form/AddTeamMemberForm.tsx

```
@@ -0,0 +1,70 @@
+  },
+  {
+    id: 6,
+    nickname: '망곰곰곰맨',
+    imageUrl: 'https://avatars.githubusercontent.com/u/2345678?v=4',
+  },
+]
+export const AddTeamMemberForm = (): JSX.Element => {
+  const methods = useForm()
+  const { handleSubmit, control, watch } = methods
+  const values = watch()
+  const onSubmit = (data: any) => console.log(data)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)